### DISCUSSION TITLE
Turning pages instead of reading continuously

### DISCUSSION THREAD

**Discussion by ggHoek** [lumetrium/obsidian-teleprompter#5] [Ideas]
Love the plugin. I have been searching and waiting for months something like this! Nevertheless, I'm a weekly presenter and I use Obsidian for my notes. But I rather change pages using my clicker or something similar. Is there a way to do it? :)

**Comment by lumetrium** [comment]
Great to hear that you enjoyed the plugin! Currently, there isn't a direct way to achieve that, but your suggestion aligns with feedback I've been getting on Reddit and the Obsidian forum. It's a fantastic idea, and I'm already considering it for future updates. I'll make sure to include it in the roadmap.

In the meantime, have you considered whether the core Slides plugin might be a better fit for your case? There's also a community plugin called Advanced Slides that could be worth checking out too. It might just be the solution you're looking for.

**Reply by ggHoek** [reply]
Thanks for the answer and for the recommendation! I have been trying to use Slides/Advanced Slides with no luck, since it forces adding separators manually to the text to separate "slides". It should be automatically. Still, your plugin rocks!
